### PR TITLE
Post delete feature

### PR DESCRIPTION
**Pull request by shiara02**
### OVERVIEW

-Add post delete feature
-Change ports on yml file fixing database error

### NOTES

-Modify the edit and show blade, changing the word layout to layouts (based on the merging from Karl's push to mine)
-The user can delete any post for now, deleting user's own post only will be fixed on the next PR.
-Modify make file

### SNIPPETS

image
image

after deleting:
image
image

### CODE REVIEW

**Review by hipe-francis — DISMISSED** [review]
LGTM 🚀

**Review by hipe01mark — DISMISSED** [review]
LGTM :bow:

File: src/routes/web.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
     Route::get('/post/{post}', [PostController::class, 'show'])->name('post.show');
     Route::get('/post/{post}/edit', [PostController::class, 'edit'])->name('post.edit');
     Route::put('/post/{post}', [PostController::class, 'update'])->name('post.update');
+    Route::delete('/post/{post}', [PostController::class, 'destroy'])->name('post.destroy');
```
Comment (bpocrafael) [inline src/routes/web.php]: can't we maybe put in a resource all of the post?

File: Makefile [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
-	docker-compose exec -it backend php artisan passport:install --force
-	docker-compose exec -it backend php artisan migrate:fresh --seed
-	docker-compose exec -it backend php artisan module:seed
+	docker compose exec -it backend php artisan key:generate
+	docker compose exec -it backend php artisan optimize:clear
+	docker compose exec -it backend php artisan migrate --seed
 
 optimize:
-	docker-compose exec -it backend php artisan optimize:clear
+	docker compose exec -it backend php artisan optimize:clear
```
Comment (hipe01mark) [inline Makefile]: Create separate pr for these changes.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,22 +1,22 @@
-version: '3.7'
+version: '3.9'
```
Comment (hipe01mark) [inline docker-compose.yml]: Create separate pr for these changes.

File: src/routes/web.php (hunk 1 above)
Comment (hipe01mark) [reply src/routes/web.php:20]: + 1 for this one

File: src/routes/web.php [hunk 4]
```diff
@@ -13,10 +13,7 @@
     Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
     Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
     Route::get('/home', [UserController::class, 'home'])->name('user.home');
-    Route::post('/posts', [PostController::class, 'store'])->name('post.store');
-    Route::get('/post/{post}', [PostController::class, 'show'])->name('post.show');
-    Route::get('/post/{post}/edit', [PostController::class, 'edit'])->name('post.edit');
-    Route::put('/post/{post}', [PostController::class, 'update'])->name('post.update');
+    Route::resource('post', PostController::class);
```
Comment (bpocrafael) [inline src/routes/web.php]: posts?? plural?

File: docker-compose.yml (hunk 3 above)
Comment (bpocrafael) [reply docker-compose.yml]: @shiara02 I thought this is in a separate pr??

File: Makefile (hunk 2 above)
Comment (bpocrafael) [reply Makefile]: also this one?

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]

**Pull request merged**